packages/react/testing-library/api-extractor.json

```diff
@@ -4,12 +4,21 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
   "extends": "../../../api-extractor.json",
-  "mainEntryPointFilePath": "<projectFolder>/types/index.d.ts",
+  "mainEntryPointFilePath": "<projectFolder>/types/entry.d.ts",
   "compiler": {
     "overrideTsconfig": {
       "compilerOptions": {
         "skipLibCheck": true,
       },
     },
   },
+  "bundledPackages": [
+    "@testing-library/dom",
```

Comment:
I don't think we need this. Since the only result of this is making the API report (`packages/react/testing-library/etc/react-lynx-testing-library.api.md`) a big mess (and also the API documentation, too).
```suggestion
```